sdk/turing/router/config/router_config.py

```diff
@@ -49,8 +49,8 @@ class RouterConfig:
     ensembler: Union[RouterEnsemblerConfig, Dict] = None
 
     def __init__(self,
-                 environment_name: str,
-                 name: str,
+                 environment_name: str = None,
+                 name: str = None,
```

Comment:
Introducing the default `None` argument values so users can now construct `RouterConfig` SDK classes to create new router versions (which do not require these values). In a way, `RouterConfig` is a single common interface for 3 main use cases, 1) creating new routers, 2) updating routers, and 3) creating new router versions.